src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/components/LogAnalysisRiskAndJiraModal/components/ActivityHeadingContent.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo } from 'react'
+import { Container, Text, Layout } from '@wings-software/uicore'
+import { Color } from '@harness/design-system'
+import HighchartsReact from 'highcharts-react-official'
+import Highcharts from 'highcharts'
+import { useStrings } from 'framework/strings'
+import { getEventTypeColor, getEventTypeLightColor } from '@cv/utils/CommonUtils'
+import type { ActivityHeadingContentProps } from '../LogAnalysisRiskAndJiraModal.types'
+import { getChartsConfigForDrawer } from '../LogAnalysisRiskAndJiraModal.utils'
+import { getEventTypeFromClusterType } from '../../../LogAnalysisRow.utils'
+import css from '../LogAnalysisRiskAndJiraModal.module.scss'
+import logRowStyle from '../../../LogAnalysisRow.module.scss'
+
+export function ActivityHeadingContent(props: ActivityHeadingContentProps): JSX.Element {
+  const { count, trendData, activityType } = props
+  const { getString } = useStrings()
+
+  const chartsConfig = useMemo(() => getChartsConfigForDrawer(getString, trendData?.series), [trendData])
```

Comment:
Codacy has a fix for the issue: [React Hook useMemo has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232318)
```suggestion
  const chartsConfig = useMemo(() => getChartsConfigForDrawer(getString, trendData?.series), [getString, trendData?.series])
```